### PR TITLE
LPS-157076: Revert ERC tests usign UUID and fix service part

### PR DESCRIPTION
**Pull request by sergiojimcos**
**What is new?**

- Update `AssetCategoryLocalServiceTest` and `AssetVocabularyServiceTest` without using `externalRefereceCode`
- Update `BlogEntryLocalServiceTest` without using `externalRefereceCode` and `BlogsEnryLocalService` setting an UUID if the `externalReferenceCode` is NULL.
- Update `DLAppServiceWhenAddingAFileEntryTest` and `DLFileEntryLocalServiceTest` without using `externalRefereceCode`.
- Update `JournlFolderServiceTest` without using `externalRefereceCode` and `JournalFolderLocalService` setting an UUID if the `externalReferenceCode` is NULL.
- Update `KBArticleLocalServiceTest` and `KBFolderLocalServiceTest` without using `externalRefereceCode`.
- Update `MBMessageLocalServiceTest`  without using `externalRefereceCode`.
- Update `WikiNodeLocalServiceTest` and `WikiPageLocalServiceTest` without using `externalRefereceCode`

**Note:** Just reverting all these tests that have been deleted: 5b97d89eb16a79a073e4db6d9522069545200107

**JIRA Ticket**
https://issues.liferay.com/browse/LPS-157076

### CODE REVIEW

**Comment by sergiojimcos** [comment]
ci:test:relevant

File: modules/apps/asset/asset-test/src/testIntegration/java/com/liferay/asset/service/test/AssetCategoryLocalServiceTest.java [hunk 1]
```diff
@@ -119,6 +119,28 @@ public void testAddCategoryWithExternalReferenceCode() throws Exception {
...
 
+	@Test
+	public void testAddCategoryWithoutExternalReferenceCode() throws Exception {
+		AssetVocabulary assetVocabulary = AssetTestUtil.addVocabulary(
+			_group.getGroupId());
+
+		AssetCategory assetCategory = AssetTestUtil.addCategory(
+			_group.getGroupId(), assetVocabulary.getVocabularyId());
+
+		String externalReferenceCode = String.valueOf(assetCategory.getUuid());
```
Comment (javierdearcos) [inline modules/apps/asset/asset-test/src/testIntegration/java/com/liferay/asset/service/test/AssetCategoryLocalServiceTest.java:130]: The UUID is already a String so I think you don't need the String.valueOf call

**Review by javierdearcos — CHANGES_REQUESTED** [review]
I found some things to improve:

I think the String.valueOf(...getUuid()) is not needed in any entity as the UUID it is already a string.

Also, I see that we have two patterns to check the default externalReferenceCode.

One pattern is to just create the object and check the that the returning externalReferenceCode is the UUID

Another pattern is to create the object, check the returning externalReferenceCode, and then get the object by externalReferenceCode and check is the same object.

I would go with the second approach in all cases as we are testing deeper the functionality.

Last thing. I think it is a good practice when you revert a commit to leave it as a revert explaining the reason, instead of squashing it with new changes

**Comment by sergiojimcos** [comment]
ci:test:relevant

**Review by javierdearcos — APPROVED** [review]
Perfect 🎉

**Review by ccorreagg — APPROVED** [review]

**Comment by susanchen3** [comment]
Reviewed. Test failures are unrelated :heavy_check_mark: Looks good to manually forward from a QA perspective! :)

**Comment by sergiojimcos** [comment]
ci:forward

**Comment by sergiojimcos** [comment]
Manually forwarding: https://github.com/brianchandotcom/liferay-portal/pull/119662

**Pull request closed without merging**